src/servlets/utils/helpers.ts

```diff
@@ -7,7 +7,7 @@ import {
 } from './constants'
 
 export const getTrack = async (id: number): Promise<any> => {
-  const t = await libs.Track.getTracks(1, 0, [id])
+  const t = await libs.Track.getTracks(1, 0, [id], null, null, null, null, /* withUsers */ true)
```

Comment:
👀

Reply:
very gnarly. i can add the other /* comments */ to make it easier on the "eyes"